.github/workflows/deploy-pypi.yml

```diff
@@ -9,17 +9,17 @@
 name: Upload Python Package
 
 on:
+  workflow_dispatch:
   pull_request:
     types: closed
     branches:
       - main
-      - release**
     paths:
       - 'gradio/version.txt'
 
 jobs:
   deploy:
-    if: github.event.pull_request.merged == true
+    if: github.event.pull_request.merged == true || github.event.action == ''
```

Comment:
What happens if you run this without updating the version.txt file? 

My guess is that the pypi upload would fail (since there already is a package with the current version) but I'm worried we're going to overwrite the front-end assets in the cdn.

Reply:
Ah nice catch. I will just have it terminate if an existing version is found on aws

Reply:
Sounds good! Might be good to just terminate as soon as the job starts if there's an existing version on pypi.

pypi was a json api here: https://pypi.org/pypi/gradio/json

and theres a `releases` key that has all the known releases. We could write a script to check the current `version.txt` with the pypi release history.

###Example output of api:

![image](https://user-images.githubusercontent.com/41651716/188723650-209253dd-1152-4b9c-ae43-6d37f1877427.png)

Reply:
Should be fixed now

Reply:
Good point about pypi, that's better than checking S3. Let me fix that up